checkov/terraform/checks/graph_checks/aws/RDSEnableCopyTagsToSnapshot.yaml

```
@@ -0,0 +1,40 @@
+metadata:
+  id: "CKV2_AWS_60"
+  name: "Ensure RDS instance with copy tags to snapshots is enabled"
+  category: "GENERAL_SECURITY"
+definition:
+  and:
+    - cond_type: attribute
+      resource_types: "aws_db_instance"
+      attribute: "copy_tags_to_snapshot"
+      operator: "jsonpath_equals"
+      value: "true"
+
+    - cond_type: attribute
+      resource_types: "aws_db_instance"
+      attribute: "copy_tags_to_snapshot"
+      operator: "jsonpath_exists"
+
+    - cond_type: attribute
+      resource_types: "aws_db_instance"
+      attribute: "copy_tags_to_snapshot"
+      operator: "jsonpath_exists"
+
+    - cond_type: attribute
+      resource_types: "aws_db_instance"
+      attribute: "engine"
+      operator: "jsonpath_not_equals"
+      value: "neptune"
+
+    - cond_type: attribute
+      resource_types: "aws_db_instance"
+      attribute: "engine"
+      operator: "jsonpath_not_equals"
+      value: "aurora"
+
+    - cond_type: attribute
+      resource_types: "aws_db_instance"
+      attribute: "engine"
+      operator: "jsonpath_not_equals"
+      value: "docdb"
```

```suggestion
    - cond_type: attribute
      resource_types: "aws_db_instance"
      attribute: "engine"
      operator: "not_within"
      value:
        - "neptune"
        - "aurora"
        - "docdb"
```
this should make it simpler 🙂

---

Awesome, thank you. I'll make a note of it even for the upcoming policies. 🙌🏻